internal/pipe/brew/template.go

```diff
@@ -39,6 +41,36 @@ const formulaTemplate = `# typed: false
 {{ if .CustomRequire -}}
 require_relative "{{ .CustomRequire }}"
 {{ end -}}
+
+class SupportedArchRequirement < Requirement
```

Comment:
Am I right (per your comments in the linked discussion) that for this to work (and Ruby to be satisfied) this extra class would have to be in a separate file?

If so then we'd be making the assumption that the whole tap/repo is basically owned by GoReleaser, which seems like a flawed assumption?

Reply:
For us our tap is managed only by goreleaser.

Reply:
in any case, this goes to the tap itself, it does not need to be a separated file...